ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/CaseMapper.java

```diff
@@ -51,6 +53,11 @@ public CCDCase to(Claim claim) {
             .map(reviewOrderMapper::to)
             .ifPresent(builder::reviewOrder);
 
+        claim.getChannelType()
+            .map(ChannelType::name)
```

Comment:
as discussed, I'd probably suggest having a dev tech huddle to discuss whether we can start getting rid of a bit of duplication between claim/ccd models. Enums could be a low hanging fruit. But for the scope of this pr is 🆗 of course

Reply:
Definitely 👍